<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WuHanghang To Do</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTabWidget" name="selectTab">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="forgotTab">
         <attribute name="title">
          <string>Forgot</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QListWidget" name="forgotList"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Wu Hanghang</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toDoTab">
         <attribute name="title">
          <string>To Do</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="toDoList">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>Wu Hanghang</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="timeLCD">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyButton">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>toDoList</tabstop>
  <tabstop>forgotList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>modifyButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>selectTab</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
